config/default.yml

```diff
@@ -192,8 +192,10 @@ RSpec/BeforeAfterAll:
 RSpec/ChangeByZero:
   Description: Prefer negated matchers over `to change.by(0)`.
   Enabled: pending
-  VersionAdded: 2.11.0
+  VersionAdded: '2.11'
+  VersionChanged: '2.13'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ChangeByZero
+  NegatedMatcher: ''
```

Comment:
```suggestion
  NegatedMatcher: ~
```

Reply:
This denotes `nil`

Reply:
`~` is disappears with rake default task.
I think some linting might be writing this back.

Reply:
I would like to take a look at lib/rubocop/rspec/config_formatter.rb a bit.

Reply:
Merged that fix, thank you!